test/functional/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml

```diff
@@ -1462,6 +1465,88 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<output type="success" caseSensitive="yes" regex="no">has been destroyed</output>
 		<output type="success" caseSensitive="yes" regex="no">is destroyed</output>
 
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>d
+
+	<test id="Test 68-a: Run ShareOrphanTest in default mode" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$ $CP_UTILS$ $PROGRAM_ORPHANTEST$</command>
+		<output type="success" caseSensitive="yes" regex="no">ShareOrphanTest Finished</output>
+
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 68-b: Make sure CustomClassLoader (not using SCC Java API) did not store classes in SCC" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$,printStats=romclass+orphan</command>
+		<output type="success" caseSensitive="yes" regex="no">Current statistics for cache</output>
+
+		<output type="failure" caseSensitive="no" regex="no">UnsafeBootClass</output>
+		<output type="failure" caseSensitive="no" regex="no">AnonClassAndUnsafeClassTest</output>
+		<output type="failure" caseSensitive="no" regex="no">Unhandled Exception</output>
+		<output type="failure" caseSensitive="yes" regex="no">Exception:</output>
+		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
+		<output type="failure" caseSensitive="yes" regex="no">Processing dump event</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Java dump</output>
+		<output type="failure" caseSensitive="yes" regex="no">JVM requested Snap dump</output>
+	</test>
+
+	<test id="Test 69-a: Rerun the test with -XX:+ShareOrphan" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ $currentMode$ -XX:+ShareOrphan $CP_UTILS$ $PROGRAM_ORPHANTEST$</command>
```

Comment:
You could add `-XX:-ShareOrphan` here before `-XX:+ShareOrphan`, which won't change the test, but give a little testing for the option.

Reply:
Fixed.